src/pages/projects/form/ProjectFormPanel1.tsx

```diff
@@ -126,53 +124,8 @@ export const ProjectFormPanel1 = ({ currentProject, suggestedProjectManager }: P
           </Field>
         </Box>
       </Box>
-
-      <Typography variant="h3" gutterBottom>
-        {t('project.project_participants')}
-      </Typography>
-      <Box
-        sx={{
-          display: 'flex',
-          flexDirection: 'column',
-          rowGap: { xs: '1.25rem', sm: '0.5rem' },
-        }}>
-        {suggestedProjectManager && (
-          <Typography>{t('project.project_manager_from_nfr', { name: suggestedProjectManager })}</Typography>
-        )}
-        <FieldArray name={ProjectFieldName.Contributors}>
-          {({ name, push, remove }: FieldArrayRenderProps) => (
-            <>
-              {values.contributors.map((contributor, index) => {
-                const thisContributor =
-                  contributor.identity.id &&
-                  currentProject?.contributors[index] &&
-                  contributor.identity.id === currentProject.contributors[index].identity.id
-                    ? currentProject.contributors[index]
-                    : undefined;
-                return (
-                  <ProjectContributorRow
-                    key={index}
-                    contributorIndex={index}
-                    baseFieldName={`${name}[${index}]`}
-                    contributor={thisContributor}
-                    removeContributor={isProjectManager(contributor) ? undefined : () => remove(index)}
-                    isRekProject={thisIsRekProject}
-                  />
-                );
-              })}
-              <Button
-                startIcon={<AddCircleIcon />}
-                onClick={() => push(emptyProjectContributor)}
-                sx={{ width: 'fit-content' }}
-                data-testid={dataTestId.registrationWizard.description.projectForm.addParticipantButton}>
-                {t('common.add')}
-              </Button>
-            </>
-          )}
-        </FieldArray>
-
-        <ProjectFundingsField currentFundings={values.funding} />
-      </Box>
+      <ProjectContributors currentProject={currentProject} suggestedProjectManager={suggestedProjectManager} />
+      <ProjectFundingsField currentFundings={values.funding} />
```

Comment:
tror du noe av dette vil kunne være gjenbrukbart for project wizard også?

Reply:
Ja heldigvis!